docs/how_it_works/univariate_drift_detection.rst

```diff
@@ -108,6 +108,36 @@ Wasserstein distance between the two distributions is given by:
 .. math::
     W_1\left(X_i^{ref},X_i^{ana}\right) = \int_\mathbb{R}\left|\hat{F}_{ref}(x)-\hat{F}_{ana}(x)\right|dx
 
+.. _univariate-drift-detection-cont-hellinger:
+
+Hellinger Distance
+........................
+
+The Hellinger Distance, is a distance metric used to quantify the similarity between two distributions. It measures the overlap between the probabilities assigned 
+to the same event by both reference and analysis. It ranges from 0 to 1 where a value of 1 is only achieved when reference assigns zero probability to each event to which 
+analysis assigns some positive probability and vice versa. 
+The formula is given by:
+
+.. math::
+    H\left(X_i^{ref},X_i^{ana}\right) = \frac{1}{\sqrt{2}}\left[\int_{}\left(\sqrt{{F}_{ref}(x)}-\sqrt{{F}_{ana}(x)}\right)^2dx\right]^{1/2}
+
+The implementation splits a continuous feature into bins, calculates the relative frequency
```

Comment:
IT would be good to improve the language a bit here, for example:
```suggestion
In order to Calculate Hellinger Distance NannyML splits a continuous feature into bins, calculates the relative frequency
```